content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -277,6 +277,11 @@ For Canal to work correctly, `--pod-network-cidr=10.244.0.0/16` has to be passed
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/rbac.yaml
 kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/canal.yaml
 ```
+{{% /tab %}}
+
+{{% tabe name="Cilium" %}}
```

Comment:
Netlify failed with this error:
```
11:28:05 PM: ERROR 2018/08/10 15:28:05 Unable to locate template for shortcode "tabe" in page "docs/setup/independent/create-cluster-kubeadm.md"
```

I think it's due to a typo here: `s/tabe/tab`